web/src/views/DandisetLandingView/ContactDialog.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-menu
+    offset-y
+    left
+  >
+    <template
+      #activator="{ on, attrs }"
+    >
+      <v-btn
+        id="download"
+        outlined
+        block
+        v-bind="attrs"
+        v-on="on"
+      >
+        <v-icon
+          color="primary"
+          left
+        >
+          mdi-card-account-mail
+        </v-icon>
+        <span>Contact</span>
+        <v-spacer />
+        <v-icon right>
+          mdi-chevron-down
+        </v-icon>
+      </v-btn>
+    </template>
+    <v-card>
+      <v-card-title class="pb-0">
+        Select an e-mail Recipient
+      </v-card-title>
+      <v-list>
+        <v-tooltip
+          :disabled="loggedIn()"
+          open-on-hover
+          right
+        >
+          <template #activator="{ on }">
+            <v-list-item
+              v-on="on"
+              :class="loggedIn() ? 'black--text' : 'grey--text'"
+              :selectable="!loggedIn()"
+              :href="`mailto:${owners}?subject=Regarding%20Dandiset%20${currentDandiset?.name}%20${currentDandiset?.metadata?.id} &body=${currentUser?.value?.username}`"
+            >
+            <v-icon
+              color="primary"
+              left
+              small
+            >
+              mdi-card-account-mail
+            </v-icon>
+              Dandiset Owner
```

Comment:
```suggestion
              Dandiset Owners
```

Reply:
changed in 5f5b0cff9887ebefcd1e5640f1faccf5b9552afc